maas/plugins/galera_check.py

```diff
@@ -90,6 +90,22 @@ def print_metrics(replica_status):
            replica_status['Threads_connected'], 'connections')
     metric('mysql_max_seen_connections', 'int64',
            replica_status['Max_used_connections'], 'connections')
+    metric('num_of_open_files', 'int64',
```

Comment:
I honestly don't think this is needed. All of the mysql status variables are already loaded into a list here:
https://github.com/rcbops/rpc-openstack/blob/master/maas/plugins/galera_check.py#L111-L112